WordPress/gutenberg
# Running npm run format:php may result in an error

If a Composer package introduces a new `PHPCS` sniff that's used by Gutenberg, executing the `npm run format:php` command may produce an error.

Thus, it's crucial to update Composer packages before running `npm run format:php`.
Props to `@michalczaplinski` for identifying this issue.
Fixes #54163.



## What?
 - [x] introduce a new `npm run preformat:php` command, which ensures that Composer packages are updated before executing `npm run format:php`;
 - [x] refactor `npm run preformat:php` and `npm run prelint:php` to avoid code duplication.

## Why?
The issue has been previosly resolved for the `npm run lint:php` command, but not for the `npm run format:php` command.
This PR ensures that both `npm run format:php` and `npm run lint:php` function consistently.

## How?
1. Introduces the new `npm run preformat:php` command.
2. Introduces the new `npm run other:update-packages:php` command (it's used internally by both `npm run preformat:php` and `npm run prelint:php`).

## Testing Instructions
1. Run `npx wp-env start` (if `wp-env` isn't installed).
3. Execute `npm run lint:php`.
4. Execute `npm run format:php`.
5. Ensure there are no errors displayed in the console.
